v4/export/dump.go

```diff
@@ -209,6 +209,12 @@ func (d *Dumper) Dump() (dumpErr error) {
 		}
 	})
 
+	// get estimate total count
+	err = d.getEstimateTotalRowsCount(tctx, metaConn)
+	if err != nil {
+		return err
```

Comment:
I think this isn't a fatal error. We can just `log.Error` here.

Reply:
fixed